.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+name: OpenWISP Utils CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=2.2
```

Comment:
It would be better to drop `django~=2.2` now.

Reply:
2.2 is LTS with support till April 2022, check here: https://www.djangoproject.com/download/#supported-versions
Many users will stick to 2.2 for a long time, I think we should drop it in 2022 only! :smile:

Reply:
Update on this: https://github.com/openwisp/openwisp-radius/pull/175#issuecomment-770219326